README.md

```diff
@@ -99,3 +99,14 @@ GITHUB_APP_PEM_BLOB=LS0tLS1...
 GITHUB_OAUTH_CLIENT_ID=xxxxxxxxxx
 GITHUB_OAUTH_CLIENT_SECRET=xxxxxxxxxx
 ```
+
+## Docker
```

Comment:
This should go into the "Quick Start" section that talks about the two existing ways to run the app: directly or via GCP Cloud Function.